subworkflows/OrganelleFinder/modules/makeblastdb.nf

```diff
@@ -0,0 +1,26 @@
+process MAKEBLASTDB {
+
+    conda "${task.ext.enable_conda ? 'bioconda::blast:2.12.0' : '' }"
+    container "${workflow.containerEngine == 'singularity' &&
+                  !task.ext.singularity_pull_docker_container ?
+              'https://depot.galaxyproject.org/singularity/blast:2.12.0--pl5262h3289130_0' :
+              'quay.io/biocontainers/blast:2.12.0--pl5262h3289130_0' }"
+
+    label 'blast'
+
+    input:
+    path genome
+    val state
+
+    output:
+    path "*.fna*"

```

Comment:
This needs to be generalised to a fasta named either `*.fasta`, `*.fa` or `*.fna`.